Microsoft.Toolkit.Uwp.UI.Controls.DataGrid/DataGrid/DataGridComboBoxColumn.cs

```diff
@@ -415,7 +421,7 @@ protected internal override void RefreshCellContent(FrameworkElement element, Br
                 }
                 else if (propertyName == DATAGRIDCOMBOBOXCOLUMN_itemsSourceName)
                 {
-                    OwningGrid.OnColumnBindingChanged(this);
```

Comment:
Why remove this line? What does it break? Thanks.

Reply:
It is actually a separate issue that probably deserves its own pull request. If you use IncrementalLoading and then update the ComboBoxColumn's ItemsSource the whole UI slows down and the more times you do it the worse it gets. I'm not sure why, but commenting out the line solves the issue.  Should I put it back in and address it separately?

Reply:
@filipwa84, yes please let's put it back and handle it separately. That sounds much better to me. Thanks.

Reply:
Ok

Reply:
@filipwa84 let me know what the new issue number is too.  Do you think this fix can make it in the 5.1.1 patch this week? @RBrid will you have time to re-review once the final comments are addressed later this week?

Reply:
@michael-hawker, yes - no problem reviewing this week.

Reply:
@michael-hawker sorry for late reply. i just pushed the requested changes, I hope it is not too late for the 5.1.1 patch.